modules/bvl_feedback/ajax/open_bvl_feedback_thread.php

```diff
@@ -13,8 +13,31 @@
 ini_set('default_charset', 'utf-8');
 require_once "bvl_panel_ajax.php";
 
-if (isset($_POST['feedbackID']) && isset($_POST['candID'])) {
-    $feedbackThread->openThread($_POST['feedbackID']);
+$feedbackid = $_POST['feedbackID'] ?? '';
+
+if (empty($feedbackid) || !is_numeric($feedbackID)) {
```

Comment:
```suggestion
if (! \Utility::valueIsPositiveInteger) {
```

This will also make sure that it's not a decimal. Not a big deal though